src/util/types.ts

```diff
@@ -27,6 +27,14 @@ export interface BaseTxInfo {
    * The checkpoint block number (u32), in hex.
    */
   blockNumber: number;
+  /**
+   * Describe the longevity of a transaction. It represents the validity from
+   * the `blockHash` field, in number of blocks. Defaults to 50 blocks, which
+   * translates to ~5 minutes.
+   *
+   * @default 50
```

Comment:
Not sure what to put as default. With 50, it's actually 64

Reply:
Let's do 64 then. I think we were using 50 because that's what Apps does, but let's not say ~5 minutes when it will actually be ~6m24s